script/testing/util/common.py

```diff
@@ -12,6 +12,12 @@
 from util import constants
 
 
+class MemoryInfo:
```

Comment:
Why do we need a class for memory Info? Its just two data attributes.

Reply:
The point here is to organize the info. I can do it in a tuple or just 2 vars, but since these 2 vars are used in multiple functions and files, it is important to keep track of which is which and what do they mean. If you think putting them into a class is a bit overdoing, I can also do them with a `namedtuple`, which serves the same purpose but a bit more flexible and succinct.

Reply:
Changed to `namedtuple`